src/data/weather.ts

```diff
@@ -164,3 +190,165 @@ const getWeatherExtrema = (
     }
   `;
 };
+
+export const getWeatherStateSVG = (state: string): SVGTemplateResult => {
+  return svg`
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 17 17"
+  >
+  ${
+    state === "sunny"
+      ? svg`
+          <path
+            fill="var(--weather-icon-sun-color, #fdd93c)"
+            d="m 14.39303,8.4033507 c 0,3.3114723 -2.684145,5.9956173 -5.9956169,5.9956173 -3.3114716,0 -5.9956168,-2.684145 -5.9956168,-5.9956173 0,-3.311471 2.6841452,-5.995617 5.9956168,-5.995617 3.3114719,0 5.9956169,2.684146 5.9956169,5.995617"
+          />
+        `
+      : ""
+  }
+  ${
+    state === "clear-night"
+      ? svg`
+          <path
+            fill="var(--weather-icon-moon-color, #fdf9cc)"
+            d="m 13.502891,11.382935 c -1.011285,1.859223 -2.976664,3.121381 -5.2405751,3.121381 -3.289929,0 -5.953329,-2.663833 -5.953329,-5.9537625 0,-2.263911 1.261724,-4.228856 3.120948,-5.240575 -0.452782,0.842738 -0.712753,1.806363 -0.712753,2.832381 0,3.289928 2.663833,5.9533275 5.9533291,5.9533275 1.026017,0 1.989641,-0.259969 2.83238,-0.712752"
+          />
+        `
+      : ""
+  }
+  ${
+    state === "partlycloudy"
+      ? svg`
+          <path
+            fill="var(--weather-icon-sun-color, #fdd93c)"
+            d="m14.981 4.2112c0 1.9244-1.56 3.4844-3.484 3.4844-1.9244 0-3.4844-1.56-3.4844-3.4844s1.56-3.484 3.4844-3.484c1.924 0 3.484 1.5596 3.484 3.484"
+          />
+        `
+      : ""
+  }
+  ${
+    cloudyStates.has(state)
+      ? svg`
+          <path
+            fill="var(--weather-icon-cloud-back-color, #d4d4d4)"
+            d="m3.8863 5.035c-0.54892 0.16898-1.04 0.46637-1.4372 0.8636-0.63077 0.63041-1.0206 1.4933-1.0206 2.455 0 1.9251 1.5589 3.4682 3.4837 3.4682h6.9688c1.9251 0 3.484-1.5981 3.484-3.5232 0-1.9251-1.5589-3.5232-3.484-3.5232h-1.0834c-0.25294-1.6916-1.6986-2.9083-3.4463-2.9083-1.7995 0-3.2805 1.4153-3.465 3.1679"
+          />
+          <path
+            fill="var(--weather-icon-cloud-front-color, #f9f9f9)"
+            d="m4.1996 7.6995c-0.33902 0.10407-0.64276 0.28787-0.88794 0.5334-0.39017 0.38982-0.63147 0.92322-0.63147 1.5176 0 1.1896 0.96414 2.1431 2.1537 2.1431h4.3071c1.1896 0 2.153-0.98742 2.153-2.1777 0-1.1896-0.96344-2.1777-2.153-2.1777h-0.66992c-0.15593-1.0449-1.0499-1.7974-2.1297-1.7974-1.112 0-2.0274 0.87524-2.1417 1.9586"
+          />
+        `
+      : ""
+  }
+  ${
+    rainStates.has(state)
+      ? svg`
+          <path
+            fill="var(--weather-icon-rain-color, #30b3ff)"
```

Comment:
we should do this with classes

Reply:
```suggestion
            class="rain"
```
```css
.rain {
  fill: var(--weather-icon-rain-color, #30b3ff);
}
```

Reply:
It needs to have a default fill anyway. I figured this would be the best way to make sure use across elements it would be easily themeable

Reply:
This makes it so we don't need to add classes to every element, card, etc that will use the svg

Reply:
Just export the styles for the same file so that they can use it.
```
export const styles = css`.rain {
  fill: var(--weather-icon-rain-color, #30b3ff);
}`
```